management/connection.go

```diff
@@ -861,6 +861,8 @@ type ConnectionOptionsAD struct {
 
 	UpstreamParams map[string]interface{} `json:"upstream_params,omitempty"`
 
+	Thumbprints *[]string `json:"thumbprints,omitempty"`
```

Comment:
This also seems to have been missed from the AD connection options type.

Reply:
To note also that the SAML conn type has thumbprints defined as `[]interface{}` instead.

Reply:
I think the SAML property was added before the schema repo so we didn't know the type of it?